src/app.rs

```diff
@@ -299,18 +267,6 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
         }
     }
 
-    pub fn patch_window_event_handlers(&self) {
```

Comment:
Remember: we drop `window_events`, making this redundant.

Reply:
Try `cargo make start subscribe` => http://localhost:8000/ => resize window => check that `Window size: ww x hh` has been changed.